src/index.d.ts

```diff
@@ -132,6 +145,22 @@ declare namespace SDKStandardComponents {
         constructor(config: BaseRequestConfigType)
     }
 
+    type putThirdpartyRequestsTransactionsRequest = {
+        transactionRequestId: string;
+        transactionRequestState: 'RECEIVED' | 'PENDING' | 'ACCEPTED' | 'REJECTED';
+    }
+
+    type postThirdpartyRequestsTransactionsAuthorizationsRequest = {
+        challenge: {
+            payload: ArrayBuffer;
```

Comment:
@kleyow why `ArrayBuffer` not a `string`? What are the benefits? if we pass `ArrayBuffer` instance for `challenge.payload` to `postThirdpartyRequestsTransactionsAuthorizations` how it will be transformed to JSON request payload?

Reply:
can you add a unit test with an example payload using `ArrayBuffer` instance for property to see that this approach is working?

Reply:
@kleyow please can you respond to @eoln on his ArrayBuffer question so I can progress this for you.

Reply:
@eoln Admittingly this is a bit of copy and paste from https://github.com/mojaloop/sdk-standard-components/pull/74/files#diff-b739f95ace6bbf2f0fd5e8cf1dab97d8R111

> why ArrayBuffer not a string?
Thinking about it now. I'm not sure why we would use `ArrayBuffer` since `sdk-standard-components` just acts as a pass through to peer services. 

>What are the benefits?

None that I can think of. Speed? Space?

> if we pass ArrayBuffer instance for challenge.payload to postThirdpartyRequestsTransactionsAuthorizations how it will be transformed to JSON request payload?

I believe just a flat string. 

@lewisdaly What was your thinking behind this type, I'm guessing since you have a better oversight of the system that you might have a justification I don't see.

Reply:
I suggest using a simple 'string'

Reply:
I have no objection to it being a string. Just waiting on Lewis' thoughts.

Reply:
Hey all, I think I was trying to be fancy here. After more research into `ArrayBuffer`, it looks like string will be more appropriate. Please change it to a string.